schemas/src/registry_schemas/schemas/name_request.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request",
+    "type": "object",
+    "title": "Name Request Schema",
+    "properties": {
+        "nrNumber": {
+            "type": "string"
+        },
+        "nrName": {
+            "type": "string"
+        },
+        "legalType": {
+            "type": "string",
+            "title": "The Legal Type of the business.",
+            "default": "BC",
+            "enum": [
+                "BC",
+                "CP"
+            ]
+        },
+        "requestType": {
+            "type": "string"
+        },
+        "status": {
+            "type": "string"
+        },
+        "expDate": {
+            "type": "string",
+            "format": "date"
+        },
+        "condition": {
+            "type": "string"
+        },
+        "submittedBy": {
```

Comment:
Submitter info (aka applicant info) will need to be full contact-style details, ie: first name, last name, address fields, etc. I just spoke to Scott and he envisions being able to populate the Business Contact Info from these fields in the NR. So *if* we want to save this data and not have to retrieve the NR from API call to get this info, we'll want all those fields in this schema. 

Here's a link to the model in the namex project that this data will come from: https://github.com/bcgov/namex/blob/master/api/namex/models/applicant.py

Reply:
Awesome! This is a great thought, there is certainly some questions still around the process a little. Will discuss in person :)